components/brave_wallet/browser/password_encryptor_unittest.cc

```diff
@@ -16,7 +17,7 @@ namespace brave_wallet {
 
 namespace {
 base::span<const uint8_t> ToSpan(std::string_view sv) {
-  return base::as_bytes(base::make_span(sv));
+  return base::as_byte_span(sv);
```

Comment:
This function is just obfuscating thing. I think it should be removed.

Reply:
https://github.com/brave/brave-core/pull/26431/commits/6809115eb5276f7e5f97305fc0e0189ade29d515